frontend/src/js/config/routes.tsx

```diff
@@ -42,7 +43,8 @@ export const routeConfigs: RouteConfigs = {
   releases: { path: 'releases', element: <Releases />, title: 'Releases' },
   settings: { path: 'settings', element: <Settings />, title: 'Settings' },
   signup: { path: 'signup', element: <Signup />, title: 'Tenants', isPublic: true },
-  tenants: { path: 'tenants', element: <TenantPage />, title: 'Tenants' }
+  tenants: { path: 'tenants', element: <TenantPage />, title: 'Tenants' },
+  subscription: { path: 'subscription', element: <SubscriptionPage />, title: 'Upgrade your subscription' }
```

Comment:
```suggestion
  subscription: { path: 'subscription', element: <SubscriptionPage />, title: 'Upgrade your subscription' },
  tenants: { path: 'tenants', element: <TenantPage />, title: 'Tenants' }
```
just to keep things sorted there.

and: 🤫 the commit order isn't fully supportive of the `<SubscriptionPage />` introduction here 😉